src/arch/xtensa/include/arch/debug/panic.h

```diff
@@ -40,7 +40,8 @@ static inline void fill_core_dump(struct sof_ipc_dsp_oops_xtensa *oops,
 				     + sizeof(struct sof_ipc_panic_info);
 	oops->plat_hdr.stackptr = stack_ptr;
 
-	oops->epc1 = *epc1;
+	if (epc1)
+		oops->epc1 = *epc1;
```

Comment:
will this be clear by default ? i.e. so we know there is no exception ?

Reply:
@lgirdwood It's read and passed only in exception handler. In panic called manually it's alwyas NULL.